book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Your method of implementing this method is wrong. Since this is the book class it should take in person and date as arguments and create a rental with it.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```